org.scala-ide.sdt.core/src/org/scalaide/ui/internal/preferences/CallByNameParamCreationPreferencePage.scala

```diff
@@ -0,0 +1,44 @@
+package org.scalaide.ui.internal.preferences
+
+import org.eclipse.swt.widgets.Composite
+import org.eclipse.swt.widgets.Control
+import org.eclipse.core.runtime.preferences.AbstractPreferenceInitializer
+import org.scalaide.core.IScalaPlugin
+
+class CallByNameParamCreationPreferencePage extends BasicFieldEditorPreferencePage("Configure highlighting for the creation of call-by-name parameters") {
+  import CallByNameParamCreationPreferencePage._
+
+  override def createContents(parent: Composite): Control = {
+    val control = super.createContents(parent).asInstanceOf[Composite]
+    PrefPageUtils.mkLinkToAnnotationsPref(parent)(a => s"More options for highlighting for call-by-name parameters on the $a preference page.")
+    control
+  }
+
+  override def createFieldEditors() {
+    addBooleanFieldEditors(
+      P_ACTIVE -> "Enabled",
+      P_BOLD -> "Bold",
+      P_ITALIC -> "Italic",
+      P_FIRST_LINE_ONLY -> "Only highlight the first line")
+  }
+}
+
+object CallByNameParamCreationPreferencePage {
+  val BASE = "scala.tools.eclipse.ui.preferences.callByNameParamCreation"
```

Comment:
In the past scala-ide contributors agreed to use CamelCase (if I remember correctly) and to inline constant names due to better grep'ability. Don't rely too much on the design of older code - a lot of it is broken and needs to be refactored.